02/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const linksList = document.querySelectorAll("a[data-url]");
+
+if (linksList) {
```

Comment:
Tutaj `if` jest niepotrzebny bo `querySelectorAll` zwraca obiekt "tablicopodobny", co powoduje, że to zawsze będzie prawda - obojętnie czy zostały odnalezione elementy czy nie. Jak już bardzo chcemy użyć if-a to tak: `if(linksList.length > 0)`